web/source/kmwdomevents.ts

```diff
@@ -948,8 +951,8 @@ class DOMTouchHandlers extends DOMEventHandlers {
       target?: EventTarget;
     };
 
-    if(e instanceof TouchEvent) {
-        tEvent=e.touches[0];
+    if(Util.instanceof(e, "TouchEvent")) {
```

Comment:
One thing we need to be careful of here is the current context - for an event on an element in an `iframe`, the context would be the iframe's `window` and not the parent document's `window`. We can still do `e instanceof e.target.ownerDocument.defaultView.MouseEvent` though, I think.

Reply:
?  How is this not mindful of the current context?  That's exactly what `Util.instanceof` is designed to address.